### PR TITLE
add screen reader styles to full and skeleton stylesheets

### PR DESCRIPTION
**Pull request by trishasalas**
https://central.tri.be/issues/62500

### CODE REVIEW

File: src/resources/css/tribe-events-full.css [hunk 1]
```diff
@@ -1968,6 +1968,18 @@ body.tribe-theme-twentysixteen .datepicker table {
 /* = Events Retina
 =============================================*/
 
+/* Utility Classes */
+
+.screen-reader-text {
```
Comment (bordoni) [inline src/resources/css/tribe-events-full.css]: Can we make it namespaced?

Comment (trishasalas) [inline src/resources/css/tribe-events-full.css]: what do you mean by namespaced in this context?

Comment (bordoni) [inline src/resources/css/tribe-events-full.css]: `.tribe-screen-reader-text` to avoid conflicts with other people using this.

Comment (trishasalas) [inline src/resources/css/tribe-events-full.css]: it's such a common class that the only thing I could see it doing would be duplication.  It comes from the html5 boilerplate and is used in some of the more recent core themes.  I'm not opposed but I'm not sure I see the need.

File: src/resources/postcss/tribe-events-full.pcss [hunk 2]
```diff
@@ -2071,3 +2071,15 @@ body.tribe-theme-twentysixteen {
 		}
 	}
 }
+
+/* Utility Classes */
+
+.screen-reader-text {
+
```
Comment (barryhughes) [inline src/resources/postcss/tribe-events-full.pcss]: Remove this extra line?

File: src/resources/postcss/tribe-events-skeleton.pcss [hunk 3]
```diff
@@ -829,3 +829,15 @@ select.tribe-events-dropdown {
 .tribe-theme-twentysixteen table.tribe-events-calendar {
 	table-layout: auto;
 }
+
+/* Utility Classes */
+
+.screen-reader-text {
+
```
Comment (barryhughes) [inline src/resources/postcss/tribe-events-skeleton.pcss]: Remove this extra line?

**Comment by barryhughes** [comment]
Needs whitespace fixes only and looks good to go!

**Comment by barryhughes** [comment]
@trishasalas - did we get a changelog entry in place already (or can we add one as part of the PR if not)?

**Pull request merged**